src/main/java/DoWhileLoop.java

```diff
@@ -0,0 +1,16 @@
+import java.util.Scanner;
+
+public class DoWhileLoop {
+    public static void main(String[] args) {
+        int n, sum = 0, i = 0;
+        Scanner in = new Scanner(System.in);
+        System.out.println("Enter the number:");
+        n = in.nextInt();
+        do {
+            System.out.println("Sum of the numbers= " + sum);
```

Comment:
in this case the message with variable 'sum' will be printed for each iteration of loop, please move it after loop